robowflex_ompl/src/ompl_interface.cpp

```diff
@@ -102,11 +102,16 @@ void OMPL::OMPLInterfacePlanner::refreshContext(const SceneConstPtr &scene,
                                                 const planning_interface::MotionPlanRequest &request,
                                                 bool force) const
 {
-    const auto &scene_id = scene->getKey();
-    const auto &request_hash = IO::getMessageMD5(request);
+    moveit_msgs::PlanningScene scene_msg = scene->getMessage();
```

Comment:
I think keeping the ID here would be more preferable. I am afraid that recomputing the scene message every time will incur a high-cost for big scenes that are represented as octomaps.